kivy/core/window/window_sdl2.py

```diff
@@ -714,7 +714,8 @@ def request_keyboard(self, callback, target, input_type='text'):
 
     def release_keyboard(self, *largs):
         super(WindowSDL, self).release_keyboard(*largs)
-        self._win.hide_keyboard()
+        if self._system_keyboard.widget:
+            self._win.hide_keyboard()
         self._sdl_keyboard = None
```

Comment:
I don't really know why that line is there. @tito added it when he wrote the backend https://github.com/kivy/kivy/blame/master/kivy/core/window/window_sdl2.py#L717. Maybe it can be removed.